cmd/get/firmware-info.go

```diff
@@ -11,17 +11,17 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type getFirmwareFlags struct {
+type getFirmwareInfoFlags struct {
 	id string
 }
 
 var (
-	flagsDefinedGetFirmware *getFirmwareFlags
+	flagsDefinedGetFirmware *getFirmwareInfoFlags
 )
 
-// Get firmware
-var getFirmwareAvailable = &cobra.Command{
-	Use:   "firmware",
+// Get firmware info
+var getFirmwareInfo = &cobra.Command{
```

Comment:
This was more of a placeholder after the merge, so I suggest the current `get firmware`
be replaced with this, because this command is expected to return a firmware object.

The current `get firmware` returns server component data can be moved under the `get component` command, we can add a `--firmware` flag to `get command`, which will then list only the firmware attributes for a component.

Let me know if this makes sense

Reply:
I feel that `get firmware` without any additional qualifiers is ambiguous here, although I agree that this other command can be moved/renamed as well.